firefox-ios/Client/Frontend/Strings.swift

```diff
@@ -4456,6 +4456,35 @@ extension String {
         }
     }
 
+    // MARK: - Sent from Firefox / Share Link Experiment
+    public struct SentFromFirefox {
+        public struct SocialShare {
+            public static let ShareMessageA = MZLocalizedString(
+                key: "SentFromFirefox.SocialShare.ShareMessageA.Title.v134",
+                tableName: "SocialShare",
+                value: "%1$@ %2$@ Sent from %3$@ 🦊 Try the mobile browser: %4$@",
```

Comment:
@halemu if you need some reference for adding strings with parameters for sharing, this is a great example!!! :D